aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -74,6 +74,8 @@ public void testCreateTable() {
     Assert.assertTrue(response.table().parameters().containsKey(BaseMetastoreTableOperations.METADATA_LOCATION_PROP));
     Assert.assertEquals(schema.columns().size(), response.table().storageDescriptor().columns().size());
     Assert.assertEquals(partitionSpec.fields().size(), response.table().partitionKeys().size());
+    Assert.assertEquals("additionalLocations should match", tableLocationProperties.values(),
```

Comment:
Is this test passing? it fails on my side with:
Expected :[s3://xioxuan-test/writeDataLoc, s3://xioxuan-test/writeMetaDataLoc, s3://xioxuan-test/writeFolderStorageLoc]
Actual   :[s3://xioxuan-test/writeMetaDataLoc, s3://xioxuan-test/writeDataLoc, s3://xioxuan-test/writeFolderStorageLoc]

Reply:
yes I checked running the test case against master right now it passes :
```shell
pk/iceberg - (master) > ./gradlew :iceberg-aws:integrationTest --tests "org.apache.iceberg.aws.glue.TestGlueCatalogTable"

BUILD SUCCESSFUL in 1m 44s
14 actionable tasks: 1 executed, 13 up-to-date

```

Looks like the order is of the output is messed up are you testing this in your branch ? if yes some change might have caused the diff in order of keys of HashMap.

Reply:
Yes. I am testing is against master branch. Let me double check again.

Reply:
it is still failing when I run clean build on master branch. I added this fix and it works. https://github.com/apache/iceberg/pull/4355

Reply:
This is very strange, if possible can you please share the java version you are in. 

P.S thankyou for making the change to fix it, have added a feedback, do let me know what you think of it,

Reply:
openjdk version "1.8.0_322"